EDGAR/embedder.py

```diff
@@ -0,0 +1,125 @@
+"""
+Embedder class embeds phrases and returns vectors for machine learning processing
+"""
+import pandas as pd
+
+import gensim
+from gensim.models import Word2Vec
+from gensim.test.utils import common_texts
+
+import numpy as np
+
+import re
+import string
+import nltk
```

Comment:
New package?
Please update some form of README.md and/or setup.rst to update documentation on necessary packages. We cannot run this with the current packages listed.